enricher/specific/src/main/java/io/jshift/maven/enricher/specific/DockerHealthCheckEnricher.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2016 Red Hat, Inc.
+ *
+ * Red Hat licenses this file to you under the Apache License, version
+ * 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package io.jshift.maven.enricher.specific;
+
+import com.google.common.base.Objects;
+import io.fabric8.kubernetes.api.model.ContainerBuilder;
+import io.fabric8.kubernetes.api.model.ExecAction;
+import io.fabric8.kubernetes.api.model.Probe;
+import io.fabric8.kubernetes.api.model.ProbeBuilder;
+import io.jshift.kit.config.image.ImageConfiguration;
+import io.jshift.kit.config.image.build.HealthCheckConfiguration;
+import io.jshift.kit.config.image.build.HealthCheckMode;
+import io.jshift.maven.enricher.api.MavenEnricherContext;
+import io.jshift.maven.enricher.api.util.KubernetesResourceUtil;
+
+import java.util.Collections;
+import java.util.List;
+
+import static io.jshift.maven.enricher.api.util.GoTimeUtil.durationSeconds;
+
+
+/**
+ * Enrich a container with probes when health checks are defined in the {@code ImageConfiguration} of the docker maven plugin.
+ * This enricher could need a change when Dockerfile health checks will be supported natively in Openshift or Kubernetes.
+ */
+public class DockerHealthCheckEnricher extends AbstractHealthCheckEnricher {
+
+    public DockerHealthCheckEnricher(MavenEnricherContext buildContext) {
+        super(buildContext, "f8-healthcheck-docker");
```

Comment:
Change to jshift-healthcheck-docker? MAybe we should find all f8 and replace them as well. There are 14 in total in this PR